ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.1-ocp4.13-lp-interop.yaml

```diff
@@ -56,6 +56,19 @@ tests:
     cluster_profile: aws-cspi-qe
     env:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
```

Comment:
I was bit confused about which team/JIRA project some of these steps should be reported to.
For example, 
1) if there is any issue with the operator installation, should LPTOCPCI handle this?
2) Incase of infra issues in aws, should LPTOCPCI still be response?

Reply:
Yeah, we should probably discuss this in our next standup together. As long as we are consistent, I think it is fine.